CHANGELOG.md

```diff
@@ -31,7 +31,19 @@ The [1.5 release](docs/1_5_Upgrade.md) is scheduled for early July.
 
 ### Bug Fixes
 
-- Fixed case where networks using onchain permissioning could stall when the bootnodes were not validators. [\#969](https://github.com/hyperledger/besu/pull/969)
+- Added timeout to queries.  Fixes issue where `eth_getLogs` queries that were too large or too broad 
+could cause Besu to crash. [\#986](https://github.com/hyperledger/besu/pull/986)
+- Fixed issue where networks using onchain permissioning could stall when the bootnodes were not validators. [\#969](https://github.com/hyperledger/besu/pull/969)
+
+### Known Issues 
+
+Known issues are open issues categorized as [Very High or High impact](https://wiki.hyperledger.org/display/BESU/Defect+Prioritisation+Policy).
+
+#### Previously identified known issues
+ 
+- [Eth/65 not backwards compatible](KNOWN_ISSUES.md#eth65-not-backwards-compatible)
```

Comment:
The backwards incompatibility part of ETH/65 is fixed.

Reply:
Updated